src/style.scss

```
@@ -0,0 +1,127 @@
+$mainColor: #060b35;
+$secondaryColor: #616070;
+$buttonColor: #00acdc;
+
+* {
+  box-sizing: border-box;
+  margin: 0;
```

```suggestion
  box-sizing: border-box;
```
only box sizing should be in * selector. reset margin only for elements where it is necessary